src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -147,16 +141,38 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
     }
 
     /**
-     * A stub class to throw an {@code IOException} when the save method is called.
+     * Test the handling of an {@code IOException} in a dummy command execution scenario by the
+     * Storage component. This test verifies that:<br>
+     * - The expected exception ({@code e}) is thrown<br>
+     * - The resulting error message matches the expected message ({@code expectedMessage})<br>
```

Comment:
```suggestion
     * Tests the Logic component's handling of an {@code IOException} thrown by 
     *  the Storage component. 
```

* Test -> Tests
* Shortened the description, as test descriptions need not be very detailed. The previous details were not complete anyway, as it did not mention about verifying the Model component is as expected.